src/plugins/bookPlayer/plugin.js

```diff
@@ -320,7 +309,7 @@ export class BookPlayer {
     }
 
     canPlayItem(item) {
-        if (item.Path && (item.Path.endsWith('epub'))) {
+        if (item.Path && item.Path.endsWith('epub')) {
```

Comment:
Is `item.Path` file ending always lowercase?

Reply:
I believe it should be, I have yet to come accross this issue.

Reply:
Well, I guess I know what I'll be naming some of my ebook files then. :sweat_smile:

Reply:
This match is indeed case sensitive, and won't recognize anything but lowercase `epub` (and `pdf`) as "playable".